control-plane/agents/src/bin/core/controller/registry.rs

```diff
@@ -100,6 +100,8 @@ pub(crate) struct RegistryInner<S: Store> {
     thin_args: ThinArgs,
     /// Check if the HA feature is enabled.
     ha_disabled: bool,
+    /// Max snapshots limit per volume.
```

Comment:
Should we call it `Global max snapshot limit per volume` just to be clear that we fall back to this if not passed via params?

Reply:
This parameter is removed now.
1. We have the control from the storage class parameter.
2. If storage class parameter is not passed, then the max_snapshot limit is `unlimited`
3. We can use plugin command to set the new value of the max_snapshot.